man/rgbasm.1

```diff
@@ -65,7 +69,12 @@ is not specified.
 .It Fl E , Fl \-export-all
 Export all labels, including unreferenced and local labels.
 .It Fl g Ar chars , Fl \-gfx-chars Ar chars
-Change the four characters used for gfx constants.
+Set four custom digit characters used for graphics constants as alternatives to 0123.
+Valid characters are numbers (although the values of 0123 cannot be changed), letters,
```

Comment:
Allow four characters to be used for graphics constants in addition to the default
.Sq 0 , 1 , 2 ,
and
.Sq 3 .
Valid characters are numbers other than 0 to 3, letters,